SharperUniverse.Tests/Stubs/BarExportableComponent.cs

```diff
@@ -34,6 +31,6 @@ public BarExportableComponent Import(SharperEntity entity)
 			return new BarExportableComponent(entity);
 		}
 
-		public string SystemType { get; set; } = typeof(BarExportSystem).FullName;
+		public string SystemType => typeof(BarExportSystem).FullName;
```

Comment:
this is GC abuse. Please assign to a private `string` field in the constructor so it doesn't create the `Type` object every time.